From a7254a09451c0e7aa23b79c8d9e883ac7ce4dd6d Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Fri, 9 Jun 2017 08:13:48 +0200
Subject: [PATCH 18/41] ksefltest: MAINTAINERS git tree entry is incorrect

There is a few more subdirectories needed in the git tree path for the
linux-kselftest url in order to be able to properly clone it.

Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Shuah Khan <shuahkh@osg.samsung.com>
---
 MAINTAINERS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index 85bafa901e4b..7e21165afb77 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -7245,7 +7245,7 @@ KERNEL SELFTEST FRAMEWORK
 M:	Shuah Khan <shuahkh@osg.samsung.com>
 M:	Shuah Khan <shuah@kernel.org>
 L:	linux-kselftest@vger.kernel.org
-T:	git git://git.kernel.org/pub/scm/shuah/linux-kselftest
+T:	git git://git.kernel.org/pub/scm/linux/kernel/git/shuah/linux-kselftest.git
 S:	Maintained
 F:	tools/testing/selftests
 
-- 
2.13.0

